pandas/core/frame.py

```diff
@@ -3859,6 +3864,12 @@ def lookup(self, row_labels, col_labels) -> np.ndarray:
         numpy.ndarray
             The found values.
         """
+        msg = (
+            "The 'lookup' method is deprecated and will be"
```

Comment:
add: you can use .melt and .loc as a substitute (e.g. what you are recommending in the whatsnew note)